### ISSUE TITLE
Collect metrics data on number of views to the critical pages (wiki, train tab, feedback form, account settings page)

### ISSUE THREAD

**Issue by karkir0003** [DSGT-DLP/Deep-Learning-Playground#506]
get the metrics data on number of views/impressions to the following pages:

* wiki
* train tab (separated by image, tabular, nlp, classical ml, etc)
* Feedback form
* account settings page
* login page (how many people opened up the website) --> page visits

**Comment by karkir0003** [comment]
see if we can leverage google analytics to do this (i believe we are on GA4). @farisdurrani can you confirm?

**Comment by farisdurrani** [comment]
Correct, we're using GA4. Talking about that, we need to move the GA from datasciencedlp@gmail.com to dlp@datasciencegt.org

**Comment by karkir0003** [comment]
ok. is that easy to do?
